doc/contribute/contributor_expectations.rst

```diff
@@ -93,6 +93,9 @@ PR Requirements
 - Each commit in the PR must provide a commit message following the
   :ref:`commit-guidelines`.
 
+- The PR must include a description that summarizes the changes in the PR
+  by describing what the changes are and why they are being made.
```

Comment:
there is a repetition in the sentence that I think is not ideal (...PR must include a description of the changes that describes the changes ...)

```suggestion
- The PR's description must include a summary of the changes and their rationale.
```

Reply:
updated